.editorconfig

```diff
@@ -144,6 +144,9 @@ dotnet_diagnostic.IDE0034.severity = warning
 # Modifiers are not ordered.
 dotnet_diagnostic.IDE0036.severity = warning
 
+# Use local function instead of lambda.
+dotnet_diagnostic.IDE0039.severity = none
```

Comment:
If we do decide to reduce the severity, please use 'silent' instead of 'none'. This removes it from the IDE issues list, but keeps it available if you invoke suggestions on the affected code. The can be helpful if you need to switch between the two during refactoring efforts.

Reply:
Makes sense